WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderActivityLauncher.java

```diff
@@ -255,8 +255,11 @@ public static void showReaderCommentsForResult(
         showReaderCommentsForResult(fragment, blogId, postId, null, 0, null, source);
     }
 
-    public static void showReaderCommentsForResult(Fragment fragment, long blogId, long postId, DirectOperation
+    public static void showReaderCommentsForResult(@NonNull Fragment fragment, long blogId, long postId, DirectOperation
```

Comment:
Line 249 is calling this `showReaderCommentsForResult()` function with nullable fragment. We should also add `NonNull` annotation there.

Reply:
🤦 Somehow that got removed, re-added it - thanks for the catch!